docs/DeveloperGuide.adoc

```diff
@@ -317,12 +317,23 @@ image::findPerson.png[] // ADD THIS IN
 ==== Design Considerations
 
 ===== Aspect: How to execute the command for finding a contact using Java Predicates, given that a Person has multiple tags.
-* **Alternative 1 (chosen):** Have one command that allows user to choose any note to edit.
-** Pros: Easy for user to use, without switching between different commands.
-** Cons: As line number of the relevant note needs to be indicated, many levels of checking of the user input is required, and is thus more prone to errors.
-* **Alternative 2:** Have 3 separate edit commands, one for each restaurant's note.
+Current implementation of Predicates uses the `stream` and `anyMatch` commands to check each keyword against one single word, but since a person may have multiple tags, the implementation needs to be able to check each keyword against multiple tags instead of a single tag.
+
+* **Alternative 1 (chosen):** Concatenate all the tags of a person together into a String, and check the provided tag keywords against that one String.
+** Pros: Easier to implement.
+** Cons: -
+* **Alternative 2:** For each person's unique tags, create one Tag Predicate per tag.
+* Pros: Conceptually easier to understand, as well as to draw the UML diagram.
+* Cons: Many Tag Predicate objects would have to be created. This could be messy and cause debugging to be difficult.
+
+===== Aspect: How to sort the provided keywords into different Predicates given that there are multiple fields we can search against (eg. Organization, Name, etc)
+
+* **Alternative 1 (chosen):** Have a loop to parse the keywords and sort them into different Predicate keyword lists depending on where the prefixes are located. For instance, if the user types `(ab)find t/friends colleagues o/NUS`, then the keywords `friends` and `colleagues` would to go to the Tag Predicate while the keyword `NUS` would be allocated to the Organization Predicate.
+* Pros: Easier for the user because they just have to demarcate which keywords are used for which using a single prefix.
```

Comment:
Not sure if you want to indent the pros and cons to the next level with ** instead of *